src/tests/unit/tests/electron/platform/android/__snapshots__/scan-results-to-unified-results.test.ts.snap

```diff
@@ -20,10 +20,9 @@ Array [
     },
     "identifiers": null,
     "resolution": Object {
-      "formatAsCode": null,
```

Comment:
The `formatAsCode` property is part of the "real" test, and is covered elsewhere